server/controllers/finance/reports/journal/index.js

```diff
@@ -32,15 +33,64 @@ function postingJournalExport(req, res, next) {
   });
 
   let report;
-
   try {
     report = new ReportManager(REPORT_TEMPLATE, req.session, options);
   } catch (e) {
     return next(e);
   }
 
-  const unposted = Journal.buildTransactionQuery(options, false);
-  const posted = Journal.buildTransactionQuery(options, true);
+  return Journal.journalEntryList(options)
+    .then(rows => report.render({ rows }))
+    .then((result) => {
+      if (result.headers.type === 'xlsx') {
+        res.xls(result.headers.filename, filter(result.report.rows));
+      } else {
+        res.set(result.headers).send(result.report);
+      }
+    })
+    .catch(next)
+    .done();
+}
+
+function dateFormat(date) {
+
+  if (date) {
+    const dateTime = new Date(date);
+    return  moment(dateTime).format('YYYY-MM-DD HH:mm:ss');
+  } else {
+    return null;
+  }
+}
+
+function filter(rows) {
```

Comment:
You normally expect a filter function to filter things out of an array.  All this does is map.  A better name for the function is therefore something like `convertDatesToStrings` or something descriptive of what this does.

Also, I think ESLint will get angry about all these empty lines.   Go head and clean this up according to ESLint's rules.